src/test/java/org/broadinstitute/consent/http/db/DatasetDAOTest.java

```diff
@@ -828,4 +844,15 @@ private FileStorageObject createFileStorageObject(String entityId, FileCategory
         );
         return fileStorageObjectDAO.findFileById(newFileStorageObjectId);
     }
+
+    private Dataset insertDataset() {
+        User user = createUser();
+        String name = "Name_" + RandomStringUtils.random(20, true, true);
+        Timestamp now = new Timestamp(new Date().getTime());
+        String objectId = "Object ID_" + RandomStringUtils.random(20, true, true);
```

Comment:
unrelated to this ticket, but i have never understood what objectId is - is it just a UUID? what is it used for, aside from datasetId?

Reply:
It's an old value that we don't use any more. It used to refer to a Broad sample set (i.e. `SM-1234`) that we got when it was exported from ORSP. That feature was removed from ORSP ages ago and we don't really support it. We should file a ticket to get rid of it.